Python/bigtable/write_bigtable.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import apache_beam as beam
+
+from apache_beam import Create
+from apache_beam import Map
+from apache_beam.io.gcp.bigtableio import WriteToBigTable
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+def run():
+    """
+    This Apache Beam pipeline transforms and writes sample data into Google Cloud Bigtable table.
+    It performs the following steps:
+    1. Input Data Generation:
+        - Simulated data is created as a sample input dataset.
+    2. Data Transformation to Bigtable Rows:
+        - Each input element is converted into a Bigtable DirectRow format.
+    3. Writing to Bigtable:
+        - The transformed Bigtable rows are written to a specified Bigtable instance and table.
+    """
+    class BigtableOptions(PipelineOptions):
+        @classmethod
+        def _add_argparse_args(cls, parser):
+            parser.add_argument(
+                '--project_id',
```

Comment:
Although this works, I'm not comfortable with explicitly adding project_id since it's anyway required to run dataflow job so it feels like a duplication

Reply:
Pipeline could be run with different runners, not only Dataflow, so I think these are two different parameters.